articles/load-balancer/egress-only.md

```diff
@@ -157,7 +157,7 @@ In this section, you'll create the public load balancer.
 
 ## Create virtual machine
 
-You'll create a virtual machine in this section. During creation, you'll add it to the backend pool of the internal load balancer. After the virtual machine is created, you'll add the virtual machine to the backend pool of the public load balancer.
+Create a virtual machine in this section. During creation and add it to the backend pool of the internal load balancer. After the virtual machine is created, you can add the virtual machine to the backend pool of the public load balancer.
```

Comment:
Please change to the following:

In this section, you'll create a virtual machine, and add it to the backend pool of the internal load balancer. Then, you'll add the virtual machine to the backend pool of the public load balancer.

The use of "you'll" is appropriate for our audience, and aligns with the Microsoft style guidance for tone.